packages/fxa-settings/src/components/UnitRow/index.tsx

```diff
@@ -56,13 +58,13 @@ export const UnitRow = ({
       <div className="unit-row-actions">
         <div>
           {route && (
-            <a
+            <Link
               className="cta-neutral transition-standard"
               data-testid="unit-row-route"
-              href={route}
+              to={route + queryParams}
```

Comment:
Do we need to pass `queryParams` down for metrics? We'll have them stored in that `queryParams` const when the application loads.

If they don't change and we need them in our components we could also use `Context` or store this in apollo cache to read where we need them.

Reply:
> Do we need to pass queryParams down for metrics?

No, I don't believe so. I can use the `Context` value here.